radixdlt/babylon-wallet-android
# [ABW-3908] - Display discover dApps home card

[ABW-3908](https://radixdlt.atlassian.net/browse/ABW-3908)

## Description
This PR displays a new "Discover dApps" home card for new users.


## How to test

1. Create a new wallet
2. Check out the new "Discover dApps" home card
3. Click on the new home card to dismiss it and to get redirected to the ecosystem web page


[ABW-3908]: https://radixdlt.atlassian.net/browse/ABW-3908?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
